### ISSUE TITLE
Bad case:   rename *.pdf  {{f|replace(".","")}} 

### ISSUE THREAD

**Issue by derekhu** [jhotmann/node-rename-cli#27]
As mentioned in :

https://github.com/jhotmann/node-rename-cli/commit/62af80c87103fb1a7e25c7199bd9fc8c2563e637#r40840287

Bad cases:

```
rename  *.pdf   {{f|replace(".", "")}}         // expect:  "abcdefg...pdf" ==> "abcdefg.pdf"

rename *.pdf  {{f|regexReplace("\\d+", "")}}   //  expect:   'abcdefg01234.pdf' ==> "abcdegf.pdf"

// however, we will get "Invalid command". 
```

Maybe we can fix it in future:-)

**Comment by jhotmann** [comment]
Since the first one has a period in the command, you'll need to specify a file extension. You can do that with `.{{ext}}` or `.pdf`

```
rename *.pdf "{{f|replace('.','')}}.{{ext}}" -s

abcdefg...pdf → abcdefg.pdf
abcdefg01234.pdf → abcdefg01234.pdf
```

The second command works fine for me

```
rename *.pdf "{{f|regexReplace('\\d+','')}}" -s

abcdefg...pdf → abcdefg...pdf
abcdefg01234.pdf → abcdefg.pdf
```

**Comment by jhotmann** [comment]
@derekhu Sorry for the long delay, there's been a lot of life happening these past several months. I have fixed this locally so you no longer have to specify `{{ext}}`, it will be included in the 7.0.2 release.

**Comment by jhotmann** [comment]
Fixed in [7.0.2](https://github.com/jhotmann/node-rename-cli/releases/tag/7.0.2)
